commit bf4cbde13545321a658e0d10b69da87e541a1b1e
Author: eccgecko <36884529+eccgecko@users.noreply.github.com>
Date:   Wed May 23 19:22:33 2018 +0200

    Enable AES, AES bit slice, and AES NEON engines on arm64

diff --git a/arch/arm64/configs/bcmrpi3_defconfig b/arch/arm64/configs/bcmrpi3_defconfig
index e3eee7542e24..9bcb4ca898d5 100644
--- a/arch/arm64/configs/bcmrpi3_defconfig
+++ b/arch/arm64/configs/bcmrpi3_defconfig
@@ -1279,6 +1279,9 @@ CONFIG_CRYPTO_DES=y
 CONFIG_CRYPTO_LZ4=m
 CONFIG_CRYPTO_USER_API_SKCIPHER=m
 CONFIG_ARM64_CRYPTO=y
+CONFIG_CRYPTO_AES_ARM64=m
+CONFIG_CRYPTO_AES_ARM64_BS=m
+CONFIG_CRYPTO_AES_ARM64_NEON_BLK=m
 CONFIG_CRC_ITU_T=y
 CONFIG_LIBCRC32C=y
 CONFIG_MMC_BCM2835_MMC=y
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1173_bf4cbde13545321a658e0d10b69da87e541a1b1e.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1173_bf4cbde13545321a658e0d10b69da87e541a1b1e.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
